### PR TITLE
Add activity page

### PR DESCRIPTION
**Pull request by krabique**
- Add activity page with CRUD (using modal windows)
- Add user's balance in navbar

### CODE REVIEW

File: app/cells/navbar/show.slim [hunk 1]
```diff
@@ -5,8 +5,13 @@ nav.navbar.navbar-expand-md.navbar-dark.bg-danger
       span.navbar-toggler-icon
     #navbar.collapse.navbar-collapse
       ul.navbar-nav.mr-auto
+        li.nav-item.active
+          = link_to 'All transactions', transactions_path, class: 'nav-link'
```
Comment (roman-dubrovsky) [inline app/cells/navbar/show.slim]: 😱 text isn't in locales

File: app/cells/navbar_cell.rb [hunk 2]
```diff
@@ -3,24 +3,13 @@
 # The navbar cell
 class NavbarCell < ApplicationCell
   def show
+    @balance = Transaction.where(user: current_user).sum(:amount_cents)
```
Comment (roman-dubrovsky) [inline app/cells/navbar_cell.rb]: let's move this logic into separate service (will change this logic in the future)

Comment (roman-dubrovsky) [reply app/cells/navbar_cell.rb]: and don't forget about unit tests for service

File: app/views/kaminari/_first_page.html.slim [hunk 3]
```diff
@@ -0,0 +1,2 @@
+li.page-item
```
Comment (roman-dubrovsky) [inline app/views/kaminari/_first_page.html.slim]: why have you generated kaminari view into the project? Do you really need this?

File: app/controllers/transactions_controller.rb [hunk 4]
```diff
@@ -1,45 +1,54 @@
 # frozen_string_literal: true
 
 class TransactionsController < ApplicationController
```
Comment (roman-dubrovsky) [inline app/controllers/transactions_controller.rb]: let's adds some tests for `TransactionsController` controller too

File: app/views/kaminari/_first_page.html.slim (hunk 3 above)
Comment (krabique) [reply app/views/kaminari/_first_page.html.slim]: Yes, these are the styles for kaminari page buttons (bootstrap styles in this case). Without them the page buttons would look like it's early 90s! 
screen shot 2018-03-14 at 01 21 12
screen shot 2018-03-14 at 01 21 28

Comment (roman-dubrovsky) [reply app/views/kaminari/_first_page.html.slim]: can you try using a gem like https://github.com/matenia/bootstrap-kaminari-views ?

I'm expecting that you don't need to copy the markup for `kaminari`.

File: app/services/balance_service.rb [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  def balance
+    Transaction.where(user: user).sum(:amount_cents)
+      .to_money / 100
+  end
+
+  private
+
+  attr_accessor :user
+
+  def user_credentials
```
Comment (roman-dubrovsky) [inline app/services/balance_service.rb]: you forgot to remove unnecessary code and change comment above service name  😆

**Pull request merged**